runtime/Cargo.toml

```diff
@@ -4,63 +4,72 @@ edition = '2018'
 name = 'joystream-node-runtime'
 # Follow convention: https://github.com/Joystream/substrate-runtime-joystream/issues/1
 # {Authoring}.{Spec}.{Impl} of the RuntimeVersion
-version = '10.6.0'
+version = '11.20.0'
```

Comment:
I noticed recently that polkadot and substrate always increment the spec version of the runtime, never going back to 0 even when bumping the authoring version. I thought it might be best to follow the same practice. Although documentation clearly says that a validator will never attempt to produce blocks if its runtime doesn't match the authoring version, its doesn't definitely talk about regular nodes. But there is an issue perhaps with transaction replay across chains (signature checks cover only spec version not authoring version)